api/inventorypb/services.proto

```diff
@@ -153,6 +153,8 @@ message ListServicesRequest {
   // Do not use yet.
   // TODO https://jira.percona.com/browse/PMM-5112
```

Comment:
while you are here could you update comment for this field, please?

Reply:
it works now

Reply:
Updated.